### DISCUSSION TITLE
Files share by client, how to control?

### DISCUSSION THREAD

**Discussion by onght** [Ylianst/MeshCentral#6243] [Q&A]
For current with agent, the full drives is available for upload, download etc. Is there anyway to control this ? Like specify what folders to make available for sharing? Also, there is a limit on the size of files that can be edit, anyway we can change that limit?

Thanks,

**Comment by si458** [comment]
> For current with agent, the full drives is available for upload, download etc. Is there anyway to control this ?

nope
> Like specify what folders to make available for sharing?

nope
> Also, there is a limit on the size of files that can be edit, anyway we can change that limit?

nope

very simple n basic answers :)
